client/src/containers/section/sidebar/index.tsx

```diff
@@ -1,29 +1,45 @@
 'use client';
 
-import { PropsWithChildren } from 'react';
-
 import { ChevronLeft } from 'lucide-react';
 import { useRecoilValue, useSetRecoilState } from 'recoil';
 
 import { cn } from '@/lib/classnames';
 
 import { sidebarOpenAtom } from '@/store';
 
+import { Section } from '@/types/app';
+
 import { useTheme } from '@/hooks/ui/theme';
 
 import { Button } from '@/components/ui/button';
 type OpenerVariant = 'opener-dark' | 'opener-light';
 
-export default function Sidebar({ children }: PropsWithChildren) {
+export default function Sidebar({
+  children,
+  section,
+}: {
+  children: React.ReactNode;
+  section: Section;
+}) {
   const open = useRecoilValue(sidebarOpenAtom);
   const setOpen = useSetRecoilState(sidebarOpenAtom);
   const variant: OpenerVariant = useTheme('opener');
+  const maxWidth = () => {
+    if (section === 'scientific-evidence' || section === 'datasets') return '';
+    const sectionMaxWidth = {
+      'map-layers': 'max-w-[487px]',
+      practices: 'max-w-[823px]',
+      network: 'max-w-[823px]',
+    };
+    return sectionMaxWidth[section];
+  };
 
   return (
     <div
       className={cn({
-        'absolute left-[117px] top-0 flex h-full w-full max-w-[487px] flex-col bg-white shadow-lg transition-transform duration-500':
+        'absolute left-[117px] top-0 flex h-full w-full flex-col bg-white shadow-lg transition-transform duration-500':
```

Comment:
In order to be responsive, you should set the width to something like 50%. Test on smaller (desktop) screens.